Update README.md (#30086)
dotnet/sdk: README.md

## Welcome to dotnet sdk

> This repository contains core functionality needed to create .NET projects that is shared between Visual Studio and the [].NET CLI](https://learn.microsoft.com/dotnet/core/tools/).

Author: ```suggestion
This repository contains core functionality needed to create .NET projects that is shared between Visual Studio and the [.NET CLI](https://learn.microsoft.com/dotnet/core/tools/).
```

* MSBuild tasks are under [/src/Tasks/Microsoft.NET.Build.Tasks/](src/Tasks/Microsoft.NET.Build.Tasks).

See [dotnet/project-system](https://github.com/dotnet/project-system) for the project system work that is specific to Visual Studio.

> This repository also contains [common project and item templates](https://github.com/dotnet/sdk/tree/main/template_feed).

Author: ```suggestion
Common project and item templates are found in [template_feed](https://github.com/dotnet/sdk/tree/main/template_feed).
```
